share/getters/shrex.go

```diff
@@ -204,6 +205,7 @@ func (sg *ShrexGetter) GetSharesByNamespace(
 		if getErr != nil {
 			log.Debugw("nd: couldn't find peer",
 				"hash", root.String(),
+				"nid", hex.EncodeToString(id),
```

Comment:
we should eventually have method on namespace.ID to pretty print the hex right?

Reply:
It is already merged and will be there in next nmt release.